tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -2102,3 +2113,195 @@ def test_quentin_expect_column_values_to_be_in_set_auto_yes_default_profiler_con
     value_set_computed: List[int] = result.expectation_config["kwargs"]["value_set"]
 
     assert value_set_computed == value_set_expected
+
+
+@probabilistic_test
+@pytest.mark.skipif(
+    version.parse(np.version.version) < version.parse("1.21.0"),
+    reason="requires numpy version 1.21.0 or newer",
+)
+@freeze_time("09/26/2019 13:42:41")
```

Comment:
See https://github.com/great-expectations/great_expectations/pull/4364, it looks like we are planning to skip these until we have a better way of handling in CI

Reply:
@anthonyburdi Let us discuss first with @NathanFarmer -- I do not think that we need to skip them.  We can instead widen the aperture.  Otherwise, the functionality is untested.  Thanks.